README.md

```diff
@@ -224,6 +224,7 @@ Core rules are included in the `tslint` package.
   * `"parameters"` checks alignment of function parameters.
   * `"arguments"` checks alignment of function call arguments.
   * `"statements"` checks alignment of statements.
+* `"arrow-parens"` requires parentheses when using arrow function.
```

Comment:
Can we change this to "requires parentheses around the parameters of arrow function declarations."